test/org/toml/lang/annotator/TomlAnnotatorTest.kt

```diff
@@ -0,0 +1,12 @@
+package org.toml.lang.annotator
+
+import org.toml.lang.TomlTestCase
+
+class TomlAnnotatorTest : TomlTestCase() {
```

Comment:
I don't think package organization means that much in tests, as long as we don't have duplicate code. For the moment being it should be OK just to use code from `rust` package in `toml`: https://github.com/alexeykudinkin/intellij-rust/commit/3103e471f92a37109ad0d77356fbcd4d033d1510.

At the moment it is not clear, what would be an ideal package structure, because TOML may be required for cargo projects support: https://github.com/alexeykudinkin/intellij-rust/pull/38.

But yes, current layout is suboptimal :(